test/integration/smoke/test_network_permissions.py

```diff
@@ -733,6 +733,11 @@ def test_04_deploy_vm_for_other_user_and_test_vm_operations(self):
         self.exec_command("self.user_apiclient", command, expected=False)
         self.exec_command("self.otheruser_apiclient", command, expected=True)
 
+        #22. Start VM before destroying, to recreate ROOT volume that was deleted as part of restore operation
+        command = """self.virtual_machine.start({apiclient})"""
+        self.exec_command("self.user_apiclient", command, expected=False)
+        self.exec_command("self.otheruser_apiclient", command, expected=True)
+
```

Comment:
a second step number 22 ! would it make sense to split this test? It already has 26 steps , adn I see some doing double work as well.
Is this creating extra value to our testing?